scripts/globals/player.lua

```diff
@@ -191,6 +192,9 @@ tpz.player.onGameIn = function(player, firstLogin, zoning)
 
     -- remember time player zoned in (e.g., to support zone-in delays)
     player:setLocalVar("ZoneInTime", os.time())
+
+    -- Login Campaign rewards points once daily
+    player:timer(2000, function (player) tpz.events.loginCampaign.onGameIn(player) end) -- Hacky recommendation by Teo.
```

Comment:
I wish I remembered that convo coz I know there are caveats (plural) to any use of timer() but thsi is your only option I think

Reply:
yes this was discussed on the former repo PR. Until the zoneIn, onZoneIn issues get resolved this would be a hacky way to get the points =)

Reply:
roeutils has an onCharLoad function that is called, maybe something similar could be applied for this?  Also, there's the possibility of joining the two into something more generic.

Reply:
Is roeutils a lua util? (i couldnt find it) or is it use in cpp?

Reply:
We should probably add a binding for this in core. `OnGameInLate`, or similar